### ISSUE TITLE
Support AWS EFS

### ISSUE THREAD

**Issue by snakebyte91** [sonatype/nxrm3-ha-repository#71]
Would be good to have AWS EFS supported by the helm chart and to see some configuration examples.

**Comment by vswaminathan777** [comment]
@snakebyte91 The Helm charts already supports EFS for the work directory where the application logs are written. Is your request to add EFS support for the purpose of storing the blobs?

**Comment by snakebyte91** [comment]
Yes, for the purpose of storing the blobs in EFS. At the moment we test EFS and the workaround with `additionalVolumes` and `additionalVolumeMounts`
